ssl-config/src/test/resources/readme.md

```diff
@@ -0,0 +1,65 @@
+# Test Keys and Certificates
+
+This directory contains certificates and keys in various formats that are used to test the package.
```

Comment:
Just had to regenerate some of these to make the name for the server certificate `localhost` and it was a bit painful... I'll write up a script for this process and check it in, but will do so later (after this PR goes through)